content/en/docs/reference/using-api/deprecation-guide.md

```diff
@@ -25,6 +25,16 @@ deprecated API versions to newer and more stable API versions.
 
 The **v1.25** release will stop serving the following deprecated API versions:
 
+#### EndpointSlice {#endpointslice-v125}
+
+The **discovery.k8s.io/v1beta1** API version of EndpointSlice will no longer be served in v1.25.
+
+* Migrate manifests and API clients to use the **discovery.k8s.io/v1** API version, available since v1.21
+* All existing persisted objects are accessible via the new API
+* Notable changes in **endpointslices.discovery.k8s.io/v1**
+    * use per Endpoint `nodeName` and `zone` fields instead of deprecated `topology` field.
```

Comment:
suggest calling out the specific topology keys that are replaced by these dedicated fields